### ISSUE TITLE
XML tag <createSequence ...> produces invalid SQL for H2 database

### ISSUE THREAD

**Issue by hwolf** [liquibase/liquibase#1570]
We create sequence in our change sets:

```xml
<createSequence cacheSize="50"
            cycle="true"
            dataType="BIGINT"
            incrementBy="50"
            minValue="1"
            sequenceName="test_123_seq"
            startValue="1"/>
```

Starting from version 3.8.1 this tag produces invalid SQL for H2 database:

```
liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set liquibase/changelogs::xxxxx:
     Reason: liquibase.exception.DatabaseException: Syntax Fehler in SQL Befehl "CREATE SEQUENCE PUBLIC.TEST_123_SEQ AS[*] BIGINT START WITH 1 INCREMENT BY 50 MINVALUE 1 CYCLE"
Syntax error in SQL statement "CREATE SEQUENCE PUBLIC.TEST_123_SEQ AS[*] BIGINT START WITH 1 INCREMENT BY 50 MINVALUE 1 CYCLE"; SQL statement:
CREATE SEQUENCE PUBLIC.test_123_seq AS BIGINT START WITH 1 INCREMENT BY 50 MINVALUE 1 CYCLE [42000-200] [Failed SQL: (42000) CREATE SEQUENCE PUBLIC.test_123_seq AS BIGINT START WITH 1 INCREMENT BY 50 MINVALUE 1 CYCLE]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:124) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Liquibase.lambda$null$0(Liquibase.java:275) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Scope.lambda$child$0(Scope.java:160) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Scope.child(Scope.java:169) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Scope.child(Scope.java:159) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Scope.child(Scope.java:138) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Scope.child(Scope.java:222) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:274) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Scope.lambda$child$0(Scope.java:160) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Scope.child(Scope.java:169) ~[liquibase-core-4.2.0.jar:na]
	at liquibase.Scope.child(Scope.java:159) ~[liquibase-core-4.2.0.jar:na]
```

Tested with liquibase-core 3.8.1, 38.9, 3.9.0 and 4.2.0

**Comment by molivasdat** [comment]
Hi @hwolf Thank you for creating this issue and bringing this to our attention.  We will add this to the list of issues to process. What is the valid sql that this should produce?

**Comment by hwolf** [comment]
Valid SQL in this case is:
```
CREATE SEQUENCE PUBLIC.test_123_seq START WITH 1 INCREMENT BY 50 MINVALUE 1 CYCLE
```

H2 sequences have alwasy type `bigint`. See https://www.h2database.com/html/commands.html#create_sequence

**Comment by nvoxland** [comment]
This was fixed in #1992 in 4.5.0
